### DISCUSSION TITLE
Update video

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#800] [Q&A]
@ThioJoe, can you make a video update? Since the everything changed so much like:
      Remove comments using a pre-existing list or log file
      Scan a specific community post
      Multiple config files
      JSON log
      ...

Can you make a update video?

(edit: this is discussions 800)

**Comment by TechStudent10** [comment]
I don't think a main channel video going over the program for the third time is necessary, as the main program itself hasn't changed at it's core since the second video. It's mostly the same.

**Reply by mrsqueakcat** [reply]
Perhaps it's just me, but I am having an awful time setting up the program too due to problems with the Google Cloud Project side of things.  

Pages that are supposed to be there are not, or pages are oddly superimposed over each other, or half ghosted over each other, or the lefthand sidebar overlaps so much of the screen I cant see what I am working with during setup.  

Did they redesign things or are the two laptops coincidentally suffering from the exact same interface issues?

**Reply by Firecul** [reply]
> Perhaps it's just me, but I am having an awful time setting up the program too due to problems with the Google Cloud Project side of things.
> 
> Pages that are supposed to be there are not, or pages are oddly superimposed over each other, or half ghosted over each other, or the lefthand sidebar overlaps so much of the screen I cant see what I am working with during setup.
> 
> Did they redesign things or are the two laptops coincidentally suffering from the exact same interface issues?

Continuing in #802
